src/koinos/chain/system_calls.cpp

```diff
@@ -930,7 +932,14 @@ THUNK_DEFINE( verify_account_nonce_result,
                  ( "n", util::to_hex( current_nonce_value ) )( "a", util::to_base58( account ) ) );
 
   verify_account_nonce_result res;
-  res.set_value( nonce_value.uint64_value() == current_nonce_value.uint64_value() + 1 );
+
+  if( context.get_mempool_nonce() )
```

Comment:
Can we go braces or no braces here

Reply:
Yes, There was other logic in here earlier that I deleted. I am surprised clang format did not catch it...